src/lib.rs

```diff
@@ -76,3 +76,9 @@ pub struct LoadableMsOapxbcRsaKey;
     note = "Use `kanidm_hsm_crypto::structures::SealedData` instead."
 )]
 pub struct LoadableMsOapxbcSessionKey;
+
+#[deprecated(
+    since = "0.3.0",
+    note = "Use `kanidm_hsm_crypto::structures::HmacS256Key` instead."
```

Comment:
This feels unnecessary since the release docs give the information on migration, and it's now a completely separate thing.

Reply:
Where is this documented? I don't see it anywhere. The README vaguely mentions some name changes, but the Hmac changes aren't mentioned.